charts/app-of-apps/values-production.yaml

```diff
@@ -0,0 +1,59 @@
+ckanHelmValues:
+  environment: production
+  ckan:
+    args: ["gunicorn --bind 0.0.0.0:5000 wsgi:application --timeout 120"]
+    probes:
+      enabled: true
+    config:
+      site:
+        url: "https://ckan.eks.production.govuk.digital"
+        title: "data.gov.uk"
+      dbHost: "ckan-postgres.production.govuk-internal.digital"
+      redis:
+        host: "backend-redis.production.govuk-internal.digital"
+      smtp:
+        server: "email-smtp.eu-west-1.amazonaws.com:587"
+        user: "AKIASQDMCZ2ESUFDLZ6D"
```

Comment:
(no action needed) Is there a secret key that goes along with this access key ID? If it's to do later then no probs at all, just mentioning on the off chance you meant to add a secretKeyRef somewhere and forgot :)

(And just to confirm for anyone else following along, the AKIA... key id is fine to be in the clear here — it's essentially a username.)

Ideally you could authnz via IAM+IRSA and ditch the long-lived key, but don't worry about that for launch!

Reply:
Yes, there is an secret access key which needs to be added, I'll add it in

Reply:
Actually, I think its provided as part of this - https://github.com/alphagov/govuk-ckan-charts/blob/main/charts/ckan/values.yaml#L36-L38, so it should be fine as it is